src/run_sign.py

```diff
@@ -8,19 +8,22 @@
 
 import argparse
 import logging
-import os
 import sys
+from pathlib import Path
 
-from manifests.build_manifest import BuildManifest
-from sign_workflow.signer import Signer
+from sign_workflow.sign_existing_artifacts import SignExistingArtifacts
+from sign_workflow.sign_with_manifest import SignWithManifest
 from system import console
 
+ACCEPTED_SIGNATURE_FILE_TYPES = [".sig", ".asc"]
```

Comment:
The existing workflow used for maven currently is the same as it is here, which only renames the binary file to be `.asc` and does not create new `.asc` files. They are therefore essentially `.sig` binary files.
Here, I have added support so that the existing maven workflow would remain the same and we can also create `.sig` files when required (eg: opensearch)

Reply:
The issue is this change does not make properly produced `.asc` files and hides the fact they are not generated correctly.  We should either apply the workaround or not support `.asc` until it works properly

Reply:
Issue in our codebase #368

Reply:
We can work on #368 in a different PR and maybe use `gpg enarmour` as a workaround?

Reply:
[Blocking] To keep this PR mergable remove the ASC option, we can follow up on that open issue to add it back in the future.

You don't need to remove the plumbing, just the option from the array